### PR TITLE
create AUTHORS file

### PR DESCRIPTION
**Pull request by luisan00**
This is an example for a `AUTHORS` file.
Please, consider add your information or comment if there is some error or you want to discuss other file format.

If you consider that someone is not here, comment it please, once all reviewers accept the PR  the `AUTHORS` file will be alphabetized

### CODE REVIEW

File: AUTHORS [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+# - Add a great welcome to all supporters
+# 
+
+Randy Brito         rdymac              <your_user@your_domain.com>
+Luis Ruiz           luisan00            <luisan00@hotmail.com>
+
+Kevin Velasco       xkevin190
+Gustavo Grisales    gustavosinbandera1
+Jean Pierre Dudey   jeandudey
+Stephanie Nogales   stephien
```
Comment (stephien) [inline AUTHORS:13]:
```suggestion
Stephanie Nogales   stephien   <stephanie.nogales@gmail.com>
```

File: AUTHORS [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+# 
+
+Randy Brito         rdymac              <your_user@your_domain.com>
+Luis Ruiz           luisan00            <luisan00@hotmail.com>
+
+Kevin Velasco       xkevin190
+Gustavo Grisales    gustavosinbandera1
+Jean Pierre Dudey   jeandudey
+Stephanie Nogales   stephien   <stephanie.nogales@gmail.com>
+Tamara Morgan       TVMG20
```
Comment (TVMG20) [inline AUTHORS:14]:
```suggestion
Tamara Morgan       TVMG20    <morgan.tamara@hotmail.com>
```

File: AUTHORS [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+# ToDo:
+# - Add a great welcome to all supporters
+# 
+
+Randy Brito         rdymac              <your_user@your_domain.com>
+Luis Ruiz           luisan00            <luisan00@hotmail.com>
+
+Kevin Velasco       xkevin190
+Gustavo Grisales    gustavosinbandera1
+Jean Pierre Dudey   jeandudey
```
Comment (jeandudey) [inline AUTHORS:12]:
```suggestion
Jean Pierre Dudey   jeandudey    <me@jeandudey.tech>
```

**Comment by xkevin190** [comment]
my email is missing: kevinvelasco193@gmail.com

**Pull request merged**